layout

```diff
@@ -0,0 +1,246 @@
+####### EXISTING OUTLINE
+
+X=delete  G=gallery (or example)  ?=not sure where this would go
+
+    Tutorials
+?       tut: Quick Start Guide
+G       tut: Trial API: PyTorch MNIST Tutorial
+G       tut: Trial API: TensorFlow Keras Fashion MNIST Tutorial
+        tut: Accessing Data
+X       tut: Native API Tutorial
+        tut: Notebooks Tutorial
+        tut: Organizing Models in the Model Registry
+
+    How-To Guides
+        how: Install Determined CLI
+        how: Hyperparameter Tuning
+        how: Distributed Training
+        how: Environment Configuration
+        how: Data Layer
+        how: How to Install Determined
+        how: Notebooks
+        how: TensorBoard
+        how: Using Checkpoints
+X       how: IDE setup
+        how: Using the Determined REST APIs
+
+    Topic Guides
+        top: Benefits of Determined
+        top: Determined on Kubernetes
+        top: Specifying Custom Pod Specs
+        top: Effective Distributed Training
+        top: Experiment Lifecycle
+        top: Exporting Models and Checkpoints
+        top: Hyperparameter Tuning Defined
+        top: Hyperparameter Tuning With Determined
+        top: Model Definitions
+X       top: Native API
+X       top: Trial API
+        top: Best Practices for Model Definitions
+        top: Optimizing Distributed Training
+        top: Reproducibility
+        top: Scheduling
+        top: System Architecture
+        top: Determined on AWS
+        top: Determined on GCP
+        top: Elastic Infrastructure
+        top: Dynamic Agents on AWS
+        top: Dynamic Agents on GCP
+        top: Terminology and Concepts
+X       top: User Interfaces
```

Comment:
The reason we added this page was because at the time we did not reference -m anywhere and did not have any page that kind describes how our CLI is structured (outside the cli man). Just something to think about.

Reply:
Good to know.  Maybe we keep this doc or canabalize it into a couple different places then.